code/modules/mob/living/carbon/human/species_types/golems.dm

```diff
@@ -394,13 +434,19 @@
 	info_text = "As a <span class='danger'>Bananium Golem</span>, you are made for pranking. Your body emits natural honks, and you cannot hurt people when punching them. Your skin also emits bananas when damaged."
 	attack_verb = "honk"
 	attack_sound = 'sound/items/AirHorn2.ogg'
+	prefix = "Bananium"
 
 	var/last_honk = 0
 	var/honkooldown = 0
 	var/last_banana = 0
 	var/banana_cooldown = 100
 	var/active = null
 
+/datum/species/golem/bananium/random_name(gender,unique,lastname)
+	var/clown_name = pick(clown_names)
+	var/golem_name = "[uppertext(clown_name)]"
+	return golem_name

```

Comment:
but Banananium golems are just made out of Banananium, they don't HAVE to be clowny.